ui/components/ui/new-network-info/index.scss

```diff
@@ -0,0 +1,51 @@
+.new-network-info {
+  &__wrapper {
+    width: 330px;
+    height: 493px;
+    box-shadow: 0 4px 20px rgba(0, 0, 0, 0.214);
+    border-radius: 8px;
+
+    & .popover-footer {
+      border-top: none;
+    }
+
+    & .popover-header {
+      padding-bottom: 1px;
+    }
+
+    & .fa-question-circle,
+    & .popover-header__button {
+      color: var(--ui-4);
+    }
+  }
+
+  &__title {
+    height: 26px;
+  }
+
+  &__token-box {
+    align-self: center;
+  }
+
+  &__subtitle {
+    height: 17px;
+    text-align: center;
+    margin-top: 24px;
+    margin-bottom: 24px;
+  }
+
+  &__bullet-paragraph {
+    border-bottom: 1px solid var(--ui-2);
+
+    &__text {
+      width: 266px;
+    }
+  }
+
+  &__button {
+    color: var(--primary-1);
+    font-size: 0.75rem;
```

Comment:
[REQ] Use the typography mixins instead of hardcoding font-size.

Reply:
Fixed in: f70e574d84d4c8014610e427fd17aca0a8335277